docs/developer-docs/build/cdks/cdk-rs-dfinity/index.md

```diff
@@ -1,19 +1,9 @@
-# Introduction to working with Rust
+# Introduction to Developing Canisters in Rust
 
-Rust is a powerful and type sound modern programming language with an active developer community. Because Rust compiles to WebAssembly, it offers a rich development environment for writing dapps to run on the Internet Computer blockchain. To help pave the way for writing dapps in Rust that can be deployed on the Internet Computer blockchain, DFINITY provides some tools to simplify the process.
+Rust is a powerful and type sound modern programming language with an active developer community. Because Rust compiles to WebAssembly, it offers a rich development environment for writing dapps to run on the Internet Computer blockchain. To help pave the way for writing dapps in Rust that can be deployed on the Internet Computer blockchain, DFINITY provides [Rust canister development kit (Rust CDK)](https://github.com/dfinity/cdk-rs) to simplify the process.
```

Comment:
```suggestion
Rust is a powerful and type sound modern programming language with an active developer community. Because Rust compiles to WebAssembly, it offers a rich development environment for writing dapps to run on the Internet Computer blockchain. To help pave the way for writing dapps in Rust that can be deployed on the Internet Computer blockchain, DFINITY provides the [Rust canister development kit (Rust CDK)](https://github.com/dfinity/cdk-rs) to simplify the process.
```